tests/devices/qutrit_mixed/test_qutrit_mixed_sampling.py

```diff
@@ -29,7 +29,7 @@
 from pennylane.measurements import Shots
 
 
-APPROX_ATOL = 0.01
+APPROX_ATOL = 0.05
```

Comment:
Probably I'm missing something, but I thought that this wasn't necessary anymore after this change

Reply:
this is to fix intermittent failures due to random fluctuations, actually unrelated to the actual failing test cases that this PR addresses

Reply:
Ok, then it is the same issues that affected other PRs as well with opmath enabled